packages/playground/website/src/lib/site-storage.ts

```diff
@@ -22,7 +23,8 @@ const SITE_METADATA_FILENAME = 'playground-site-metadata.json';
  * NOTE: We are using different storage terms than our query API in order
  * to be more explicit about storage medium in the site metadata format.
  */
-export type SiteStorageType = 'temporary' | 'opfs' | 'local-fs';
+export const SiteStorageTypes = ['opfs', 'local-fs', 'none'] as const;
```

Comment:
I temporarily changed all the storage options naming to these three, but before this PR is merged I'd love to ensure the Query API keeps working with today's storage options names.

Reply:
It would be great if we could block merging PRs that contain changes with some special TODO pattern like `@BEFORE-MERGE`, so we could leave inline notes for ourselves and be stopped from merging if we forget.

Reply:
Actually, we could just do this with a CI test.

Reply:
I would love that!

With this particular change, though, we may not need to do that since the `storage` Query API parameter is gone. Developer documentation will need updating for sure.